scripts/accounts/gcloud/setup-principal-account.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+helpFunction()
+{
+    echo "Enrolls a Principal (end user or service account) in a project with the provided roles attached."
+    echo ""
+    echo "Arguments:"
+    echo "  -g     [Required] Google Account of an end user. Mutually exclusive with -s."
+    echo "  -s     [Required] Service Account name. Mutually exclusive with -g."
+    echo "  -p     [Required] Short project name (ID) to which the principal will be enrolled."
+    echo "  -r                Roles (basic or predefined) to be attached to the principal in the project, splitted by comma."
+    echo "  -f                Path to a file containing the roles (basic or predefined) to be attached to the principal in the project."
+    echo "  -c                Path to a YAML file containing the custom role to be attached to the principal in the project. Requires -i."
+    echo "  -i                ID to be set to the custom role provided in -c."
+    exit
+}
+
+while getopts g:s:p:r:f:c:i: flag
+do
+    case "${flag}" in
+        g) google_account=${OPTARG};;
+	s) service_account=${OPTARG};;
+        p) project_id=${OPTARG};;
+        r) roles=${OPTARG};;
+        f) roles_file=${OPTARG};;
+        c) custom_role_file=${OPTARG};;
+	i) custom_role_id=${OPTARG};;
+	h ) helpFunction; exit ;;
```

Comment:
```suggestion
	h) helpFunction ;;
```

Help function already exits